redmine_import_time_entry/static/src/js/timesheet.js

```diff
@@ -21,12 +26,10 @@ openerp.redmine_import_time_entry = function(instance) {
                     }
                 }
                 else if(new_ts.unit_amount !== 0){
-                    // Old record
-                    ts_id = new_ts.id;
-                    delete new_ts.id;
-                    res.push([1, ts_id, new_ts]);
+                    res.push([1, new_ts.id, new_ts]);
                 }
                 else{
+                    del
```

Comment:
@dufresnedavid Javascript is not really my area so I'm sorry if this is silly question but, I didn't get this line.